magda-web-client/src/UI/Pagination.js

```diff
@@ -19,7 +19,7 @@ class Pagination extends Component {
         return (
             <button
                 onClick={this.onClick.bind(this, currentIndex - 1)}
-                className="btn-prev au-btn au-btn--secondary"
```

Comment:
@chloeleichen 

Those `au-btn` classes were added for solving issue #1044 and lost during the merge I think.

It seems pagination bar won't show differences between `selected` / `unselected` button now.

![image](https://user-images.githubusercontent.com/674387/40597979-55ce0b88-6288-11e8-965b-913f0e31a517.png)

Reply:
The `au-btn` style isn't really suitable in this case since they are actually links. Added a different class `current` to identify which is the current page
![screen shot 2018-05-28 at 3 13 39 pm](https://user-images.githubusercontent.com/7508939/40598202-bb2d7814-6289-11e8-85da-72b0d47fa55a.png)

Reply:
Fixed:D thanks!